src/dune_env.mli

```diff
@@ -0,0 +1,25 @@
+open Import
+
+type stanza = Stanza.t = ..
```

Comment:
Do we need this alias?

Reply:
Yes, it's necessary unfortunately. Because we define a `Stanza` submodule below that will shadow the toplevel `Stanza` module. Maybe we can think of a better submodule name?

Reply:
Oh I see. Maybe `Conf`?